docs/ibis-for-pandas-users.ipynb

```diff
@@ -46,7 +46,7 @@
    "id": "afd136cd-18b5-48a2-8359-c47cde181c7b",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    unioned = ibis.union(t_1, t_2)
</pre><p>This should be <code>union_all</code> to match <code>pd.concat</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/6048/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='123'/>

Reply:
<p>I don't think we have a <code>union_all</code></p>

Reply:
`ibis.union` is a `UNION ALL`. `ibis.union(*args, distinct=True)` is `UNION`.

Reply:
Ah, right! Apologies, this is good to go then!